address-autocomplete/.github/workflows/deploy_prod.yml

```diff
@@ -0,0 +1,60 @@
+name: AIO App CI
```

Comment:
Did you check that it works? Seems little suspicious because there are no google creds here